web-app/src/actions/dashboard.js

```diff
@@ -0,0 +1,76 @@
+import * as actionTypes from '../actionTypes';
+import { getAllHolidays, getHolidays } from '../services/holidayService';
+import {
+  formatEventsForCalendar,
+  getEventDuration,
+} from '../utilities/dashboardEvents';
+
+/*
+  Action Creators
+*/
+
+export const selectBooking = booking => {
+  return {
+    type: actionTypes.SELECT_EVENT,
+    payload: booking,
+  };
+};
+
+export const updateEventDuration = event => {
+  return {
+    type: actionTypes.UPDATE_EVENT_DURATION,
+    payload: getEventDuration(event),
+  };
+};
+
+export const toggleBookingModal = open => {
```

Comment:
Not keen on the payload name `open`. If it's toggle then it could also be closing the modal. Maybe name it `toggle` or something similar. :)